src/main/java/core/basesyntax/service/implementations/DataProcessorImpl.java

```diff
@@ -0,0 +1,44 @@
+package core.basesyntax.service.implementations;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.service.DataProcessor;
+import core.basesyntax.service.OperationStrategy;
+import core.basesyntax.service.operationhandler.OperationHandler;
+import java.util.List;
+
+public class DataProcessorImpl implements DataProcessor {
+    private static final String REPORT_TEMPLATE = "fruit,quantity";
+    private static final int OPERATION_FIELD = 0;
+    private static final int FRUIT_NAME_FIELD = 1;
+    private static final int FRUIT_QUANTITY_FIELD = 2;
+
+    private StorageDao storageDao = new StorageDaoImpl();
```

Comment:
```suggestion
    private OperationStrategy operationStrategy;
```
you should pass instance of the strategy through a constructor